packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -20,18 +20,30 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
     }
   }
 
-  setHeadComponents([
-    <link
-      key={`gatsby-plugin-manifest-link`}
-      rel="manifest"
-      href={withPrefix(`/manifest.webmanifest`)}
-    />,
-    <meta
-      key={`gatsby-plugin-manifest-meta`}
-      name="theme-color"
-      content={pluginOptions.theme_color}
-    />,
-  ])
+  /**
+   * The user has an option to opt out of the theme_color meta tag being inserted into the head.
+   */
+  const themeColor = pluginOptions.theme_color || false
+
+  setHeadComponents(
+    [
+      <link
+        key={`gatsby-plugin-manifest-link`}
+        rel="manifest"
+        href={withPrefix(`/manifest.webmanifest`)}
+      />,
+    ].concat(
+      themeColor
+        ? [
+            <meta
+              key={`gatsby-plugin-manifest-meta`}
+              name="theme-color"
+              content={pluginOptions.theme_color}
+            />,
+          ]
+        : []
+    )
+  )
```

Comment:
Can we prep headComponents array ahead of time to avoid concat with ternary operator inside it?
```js
const headComponents = [
  <link
    key={`gatsby-plugin-manifest-link`}
    rel="manifest"
    href={withPrefix(`/manifest.webmanifest`)}
  />,
]

if (pluginOptions.theme_color) {
  headComponents.push(
    <meta
      key={`gatsby-plugin-manifest-meta`}
      name="theme-color"
      content={pluginOptions.theme_color}
    />
  )
}

setHeadComponents(headComponents)

Reply:
Yeah, I thought I got rid of those changes to that ReadMe. I am not even sure how they got changed to be honest. Dog stepping on keyboard perhaps lol.

Reply:
Is it alright that I created the headComponents array at the top of the function, and changed the other conditionals to just add to this array as we step through the function? I think this makes it pretty legible and easy to understand whats going on. I ran the tests after this change and everything still seems to work just fine.

Reply:
Oh, yeah, that's fine too 👍

Reply:
Hmm, actually this README changes (that your dog committed 🤣 ) seems like something very weird is happening and I get them locally too, I'll just merge with those - hopefully this will fix problems lol